ros/src/util/packages/autoware_rviz_plugins/rosbag_controller/src/autoware_rosbag_plugin.cpp

```diff
@@ -165,7 +165,7 @@ void Autoware_Rosbag_Plugin::on_button_record_start_clicked()
 
   /* Caculate max split size */
   if (split_size != 0 && ui->checkBox_split_size->isChecked())
-    recoParam.max_size = ceil(split_size * 1048576 * 1024);
+    recoParam.max_size = uint64_t(split_size * 1048576 * 1024);
```

Comment:
Why are you replacing a call to `ceil` with a type cast? And an unsafe one, at that.

Reply:
Sorry, I replace it for debug and forgot to change it back. Now I replace it back to `ceil`.